Changelog

```diff
@@ -1,3 +1,19 @@
+Fri Jun 8 13:51:20 CET 2018 Pascal Arlt <parlt@suse.com>
+
+	* Release v2.0.0:
+		- Removed Go 1.5 and 1.6 support 1.9 and 1.10.x are supported now
+		- Add short forms of commands to help section
+		- Update and use zypper exit codes
+		- Move from docker/engine-api to docker/docker
+		- Allow inspection of stopped containers
+		- Fix bug that caused images not to be removed properly in some cases
+		- Fix bug that caused lpc command to log to stdout
+		- Fix bug that caused force flag not to work with zypper-docker images
+		- Fix docker ps command
+		- Fix bug with zypper-docker up/patch --no-recommends
+		- Update integration tests
+		- Analyze container instead of base image by default
```

Comment:
I know we never did it for older versions, but I think it would be important to add commit SHA's on every bullet of this list (or even a link to github).

Reply:
Okay, I will add github links to each bullet referring to the respective PR's .